tv_apps/smart-system/js/app_transition_controller.js

```diff
@@ -170,18 +175,32 @@
     }
   };
 
-
   AppTransitionController.prototype._do_opening =
     function atc_do_opening() {
+      this._waitingForLoad = false;
+      var animation = this.getAnimationName('open');
+      if (animation === 'invoked') {
+        var color = (GRAY_BACKGROUND_APPS.indexOf(this.app.manifestURL) > -1)?
+                    '#E0E0E0' : 'black';
+        Animations.createCircleAnimation( this.app.element.parentNode, color)
+          .play('grow', function() {
+            // show the homescreen's fadeoverlay so we will not see the system
+            // background during fast-fade-in transition
+            homescreenWindowManager.getHomescreen().fadeOut();
```

Comment:
BTW, what's the cost to create your owned overlay?? If it's easy, please do not recycle other element to do it.